plugins/modules/iosxr_ntp_global.py

```diff
@@ -0,0 +1,1077 @@
+          type: bool
+        authentication_keys:
+          description: Authentication key for trusted time sources
+          type: list
+          elements: dict
+          suboptions:
+            id:
+              description: <1-65535>  Key number
+              type: int
+            key:
+              description: Authentication key.
+              type: str
+            encryption:
+              description: Type of key encrypted or clear-text.
+              type: bool
+        broadcastdelay:
+          type: int
+          description: Estimated round-trip delay in microseconds.
+        drift:
+          type: dict
+          description: Drift(cisco-support)
+          suboptions:
+            aging_time:
+              type: int
+              description: Aging time in hours.
+            file:
+              description: File for drift values.
+              type: str
+        interfaces:
+          type: list
+          elements: dict
+          description: Configure NTP on an interface.
+          suboptions:
+            name:
+              type: str
+              description: Name of the interface.
+            vrf:
+              type: str
+              description: Name of the vrf.
+            broadcast_client:
+              type: bool
+              description: Listen to NTP broadcasts
+            broadcast_destination:
+              type: str
+              description: Configure broadcast destination address.
+            broadcast_key:
+              type: int
+              description: Broadcast key number.
+            broadcast_version:
+              type: int
+              description: <2-4>  NTP version number.
+            multicast_key:
+              type: int
+              description: Configure multicast authentication key.
+            multicast_ttl:
+              type: int
+              description: Configure TTL to use.
+            multicast_client:
+              type: str
+              description: Configure multicast client
+            multicast_destination:
+              type: str
+              description: Configure multicast destination
+            multicast_version:
+              type: int
+              description: <2-4>  NTP version number.
+        ipv4: &ip
+          description: Mark the dscp/precedence bit for ipv4 packets.
+          type: dict
+          suboptions:
+            dscp:
+              description: Set IP DSCP (DiffServ CodePoint).Please refer vendor document for valid entries.
+              type: str
+            precedence:
+              description: Set precedence Please refer vendor document for valid entries.
+              type: str
+              choices: [ "critical", "flash", "flash-override", "immediate", "internet", "network", "priority", "routine" ]
+        ipv6: *ip
+        log_internal_sync:
+          type: bool
+          description: Logs internal synchronization changes.
+        master:
```

Comment:
For consistency with IOS and NX-OS:
```yaml
master:
  stratum: 3
```

Reply:
@ashwini-mhatre are the valid commands
`ntp master `and `ntp master 3` ?

Reply:
ack